rplugin/python3/denite/scantree.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import os
+
+
+def scantree(pathn):
+    """Use scandir if available else os.walk.
+    scandir is supposed to be much faster"""
+
+    skip_list = ['.git']
+
+    if 'scandir' in dir(os):
```

Comment:
Do

```
try:
    from os import scandir
    # scantree with scandir
except ImportError:
    # scantree without scandir
    def scantree(path):
        for root, ....
```

at the very top of the module and re-write entire you code with pre-deined `scandir`. Executing `dir(os)` everytime is not good for the performance.